codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -1178,14 +1174,14 @@ public void process() {
             }
         }
 
-        public void deactivate(final DefaultStream stream) {
+        public void deactivate(final DefaultStream stream, final Iterator<?> itr) {
```

Comment:
the class is private. all of the public methods could be made package private, but I'm not sure if it matters much. WDYT?